src/js/directives/index.js

```diff
@@ -2,7 +2,13 @@ import {app} from '../bootstrap';
 import HeaderDirective from './HeaderDirective';
 import FooterDirective from './FooterDirective';
 import UnauthorizedDirective from './UnauthorizedDirective';
+import {preview as RecipePreviewDirective, search as RecipeSearchDirective} from './RecipeDirective';
+import {view as RecipeViewDirective, liked as RecipeLikedDirective} from './RecipeDirective';
 
 app.directive('header', HeaderDirective);
 app.directive('footer', FooterDirective);
 app.directive('unauthorized', UnauthorizedDirective);
+app.directive('recipepreview', RecipePreviewDirective);
+app.directive('recipesearch', RecipeSearchDirective);
+app.directive('recipeview', RecipeViewDirective);
+app.directive('recipeliked', RecipeLikedDirective);
```

Comment:
these are used as HTML tags, no? like, `<recipeview>`? If so, most custom tags i see tend to differentiate themselves from normal, built-in tags by either using capital letters (`<RecipeView>`) or hyphens (`<recipe-view>`)